go.mod

```diff
@@ -82,7 +82,7 @@ require (
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776 // indirect
-	honnef.co/go/tools v0.1.3 // indirect
+	honnef.co/go/tools v0.1.4 // indirect
```

Comment:
Does the patch need a upgrade of deps?

Reply:
Generally I run `make check` locally and it upgrade this dep. I think we don't use `honnef.co/go/tools` in codebase actually but it includes `golangci-lint`